docs/reference/ilm/error-handling.asciidoc

```diff
@@ -0,0 +1,159 @@
+[role="xpack"]
+[[index-lifecycle-error-handling]]
+== Index Lifecycle Error Handling
+
+During Index Lifecycle Management's execution of the policy for an index, it's
+possible for a step to encounter an error during its execution. When this
+happens, ILM will move the management state into an "error" step. This halts
+further execution of the policy and gives an administrator the chance to address
+any issues with the policy, index, or cluster.
+
+An example will be helpful in illustrating this, imagine the following policy
+has been created by a user:
+
+[source,js]
+--------------------------------------------------
+PUT _ilm/policy/shrink-the-index
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "min_age": "5m",
```

Comment:
I think we should use values of `min_age` that we expect users to use in documentation. Can we use a value like 5 days here?

Reply:
Sure, no problem